### PR TITLE
Redirect sub-user to /account page after login

### PR DESCRIPTION
**Pull request by Yiin**
Fixes https://gitlab.protontech.ch/ProtonVPN/web/protonvpn-settings/issues/57

### CODE REVIEW

File: src/app/containers/LoginContainer.js [hunk 1]
```diff
@@ -2,12 +2,18 @@ import React from 'react';
 import PropTypes from 'prop-types';
 import { Link } from 'react-router-dom';
 import { c } from 'ttag';
-import { LoginForm, Href, SimpleDropdown, DropdownMenu, SignInLayout } from 'react-components';
+import { LoginForm, Href, SimpleDropdown, DropdownMenu, SignInLayout, useUser } from 'react-components';
 
 const LoginContainer = ({ stopRedirect, history, onLogin }) => {
+    const [user] = useUser();
```
Comment (mmso) [inline src/app/containers/LoginContainer.js]: You cannot use user in the public app, but you can destructure the raw user object from ...args I think

Comment (EpokK) [reply src/app/containers/LoginContainer.js]: @mmso @vincaslt Can we use the `User` returned directly by the API call then during the login?

Comment (mmso) [reply src/app/containers/LoginContainer.js]: Yeah you can destructure User from ...args

Comment (EpokK) [reply src/app/containers/LoginContainer.js]:
```js
import { isSubUser } from 'proton-shared/lib/user/helpers';
const { User = {} } = args;

if (isSubUser(User)) {
 // ...
}
```

**Pull request merged**